lib/timing.mjs

```diff
@@ -0,0 +1,125 @@
+import process from 'node:process';
+import { table } from 'table';
+
+// Inspired by ESLint's timing.js
+// https://github.com/eslint/eslint/blob/main/lib/linter/timing.js
+
+/**
+ * Start time measurement.
+ * @returns {[number, number]} Variable for tracking time.
+ */
+function startTime() {
+	return process.hrtime();
```

Comment:
[suggestion] I didn't know it until now, but `hrtime.bigint()` sounds better:

> Legacy. Use [process.hrtime.bigint()](https://nodejs.org/api/process.html#processhrtimebigint) instead.

-- https://nodejs.org/api/process.html#processhrtimetime

Reply:
if he does he will have to change l18, l28 and probably adjust the output to be rounded and/or converted.

Reply:
Thanks for your suggestion! Changed in [this commit](https://github.com/stylelint/stylelint/pull/8108/commits/9d071f7347171a3d6be33c71f5be81bb62af2796).